Solutions/Threat Intelligence/Analytic Rules/IPEntity_W3CIISLog.yaml

```diff
@@ -82,5 +82,5 @@ entityMappings:
     fieldMappings:
```

Comment:
Field Mapping is in old style

Reply:
Updated the mapping, but since there is no data to test, not splitting Name to UPNSuffix and Host to DnsName.